demo/runtime/src/runtime/session.rs

```diff
@@ -22,6 +22,8 @@ use codec::KeyedVec;
 use runtime_support::{storage, StorageVec};
 use demo_primitives::{AccountId, SessionKey, BlockNumber};
 use runtime::{system, staking, consensus};
+use runtime::system::PrivPass;
+use runtime::staking::PublicPass;
```

Comment:
these should really reside in the same module

Reply:
they now both reside in the only module that is able to actually create them.